x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/ml/text_expansion_search_rank_features.yml

```diff
@@ -0,0 +1,109 @@
+# This test uses the simple model defined in
+# TextExpansionQueryIT.java to create the token weights.
+setup:
+  - skip:
+      features: headers
```

Comment:
This was only available since 8.something. I think you need to ensure you skip older versions.

Reply:
That is correct - adding 8.8 as minimum version.

I wonder why this has not been necessary before? 🤔